pyproject.toml

```diff
@@ -1,3 +1,64 @@
+[tool.mypy]
+python_version = "3.10"
```

Comment:
could this be newer, or one should use the oldest supported version?

Reply:
I think this can be removed?

> Specifies the Python version used to parse and check the target program. The string should be in the format MAJOR.MINOR – for example 2.7. The default is the version of the Python interpreter used to run mypy.